user-manual/administer/manage-jobs.rst

```diff
@@ -219,4 +221,8 @@ users will only see jobs and be able to clear jobs which they have started. Only
 an :term:`administrator` can see the status of all jobs - however, like all users,
 administrators can only clear jobs that belong to them.
 
+.. NOTE::
+
+   As of 2.5, 
+
```

Comment:
This seems like an incomplete PR, perhaps? Note that this partial PR also ended up in your other PR!

Reply:
Weird. I must have decided to go a different direction with this and forgot to delete this. Good catch!